### PR TITLE
Replaced deprecated `pick_types` with `raw.pick`

### LINKED ISSUE #151: Move from `pick_types` to `inst.pick`
See for example: https://github.com/sappelhoff/pyprep/blob/82390acff4c15a8e8ca92ed92c46f7a50d72e049/pyprep/find_noisy_channels.py#L57

This raises:

> NOTE: pick_types() is a legacy function. New code should use inst.pick(...).

### PR DESCRIPTION
**Pull request by Ayush-Devs**
# PR Description
closes #151 
This PR:
changes all instances of `pick_types` with `raw.pick`. 
removes "import mne" from pyprep/utils.py and  pyprep/tests/test_prep_pipeline.py as they are no longer in use

# Merge Checklist
[ ] the PR has been reviewed and all comments are resolved
[ ] all [CI][what-is-ci] checks pass
[ ] (if applicable): the PR description includes the phrase `closes #<issue-number>` to automatically close an issue. [auto-close-documentation]
[ ] (if applicable): bug fixes, new features, or [API][what-is-api] changes are documented in [whats_new.rst][whats-new-file]
[ ] (if applicable): new contributors have added themselves to the authors list in the `CITATION.cff` file

### CODE REVIEW

File: pyprep/utils.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import math
 from cmath import sqrt
 
-import mne
+# import mne
```
Comment (sappelhoff) [inline pyprep/utils.py:5]: please just remove this line instead of commenting it out

**Review by sappelhoff — COMMENTED** [review]
Thanks! The tests are failing after your changes - could you look into what is causing the issue, please?

**Review by sappelhoff — COMMENTED** [review]
Hey @Ayush-Devs, just to give you a bit of feedback: I am seeing a lot of changes that are unrelated to the purpose of the issue (replaacing pick_types with pick). This makes the diff hard to read to the reviewers, and it will also open your contribution up to criticisms that you may not want. For example I see that you have removed a docstring and replaced it with a one-liner.

I strongly recommend that you keep your changes as concise and small as possible, and not to touch any lines that are unrelated.

**Comment by Ayush-Devs** [comment]
Yes sorry, I got a bit distracted in passing those tests. Thanks for the review, it will be fixed soon. 

Apart from that, I was not very confident with the amount of changes I needed to do before commiting. 
Is this normal or should I change my approach. I did try to minimize the changes but the tests kept on failing.

**Comment by sappelhoff** [comment]
> Apart from that, I was not very confident with the amount of changes I needed to do before commiting.

It is generally better to do small but meaningful commits rather than big commits that mix a lot of things

> I did try to minimize the changes but the tests kept on failing.

It is best to test locally and figure out the root cause of the failure, and then commit a small change that fixes precisely what needs fixing.

**Pull request closed without merging**